cns/service/main.go

```diff
@@ -873,6 +873,12 @@ func main() {
 			if err = httpRemoteRestService.SavePnpIDMacaddressMapping(rootCtx); err != nil {
 				logger.Errorf("Failed to fetch PnpIDMacaddress mapping: %v", err)
 			}
+
+			// Setting primary macaddress if VF is enabled on the nics for swiftv2
+			err = setVFForAccelnetNICs()
```

Comment:
we need to check if VF is enabled before calling this function?

Reply:
nope it should be a no-op for non-accelnet nics